posts/2021-08-30-sharing-data-with-fine-grained-control.md

```diff
@@ -0,0 +1,792 @@
+---
+title: "Sharing Data with Fine-grained Control"
+description: "Defining Data Visibility Constraints Among Partners in a Universal Application"
```

Comment:
Is it visibility or access?  I'd argue access since ACLs include R and W restrictions.